tests/cxx/export_nesting_1.h

```diff
@@ -0,0 +1,15 @@
+//===--- export_nesting_1.h - test input file for iwyu --------------------===//
```

Comment:
Additional includes are usually called `-d1.h` and `-i1.h`, I think for "direct" and "indirect".

So I think these two supporting headers should be renamed:
- `export_nesting_1.h` -> `export_nesting-d1.h`
- `export_nesting_2.h` -> `export_nesting-i1.h`